Gemfile

```diff
@@ -0,0 +1,15 @@
+ruby '2.2.2'
+
+gem 'activesupport', '~> 4.2.2'
+gem 'elasticsearch-model'
+gem 'activemodel'
+gem 'typhoeus'
+gem 'activerecord'
```

Comment:
Ah... I see the mention of that in the [elasticsearch-transport doc](https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-transport). Since this doesn't use Typhoeus directly, I think ideally it should allow multiple adapters the same way that gem does. It might be worth "recommending" that Typhoeus (or another library supporting persistent connections) be required in this project's README.